util/dtd/culiturgical.dtd

```diff
@@ -9,10 +9,10 @@
   <!ATTLIST anchor number CDATA #REQUIRED>
   <!ATTLIST anchor label CDATA #IMPLIED>
 <!ELEMENT footer EMPTY>
-<!ELEMENT red (#PCDATA | wide | small | footnote)*>
-<!ELEMENT wide (#PCDATA | red | small | footnote)*>
-<!ELEMENT small (#PCDATA | red | wide | footnote)*>
-<!ELEMENT footnote (#PCDATA | red | disp)*>
+<!ELEMENT red (#PCDATA | wide | small | footnote | anchor)*>
+<!ELEMENT wide (#PCDATA | red | small | footnote | anchor)*>
+<!ELEMENT small (#PCDATA | red | wide | footnote | anchor)*>
+<!ELEMENT footnote (#PCDATA | red | disp | anchor)*>
```

Comment:
Anchor inside of footnote doesn't really make any sense.

Reply:
I believe not all texts had empty footer at the end. Thus I assumed it has no functional meaning.

Page break can happen anywhere. Theoretically speaking, footnote text *can* spill to the next page. This is quite rare though and probably not present in any of the books in our corpus.

If you feel its over-engineering, just remove it.

Reply:
We'd never have a page break in the middle of a footnote in our corpus.

In the XSLT code that I had previously, the `<footer/>` acted as a flag for a second pass over the footnotes, which was how we printed the footnotes at the bottom of the page in the HTML. Maybe there is a better way?